src/Frontend/src/components/messages/MessageView.vue

```diff
@@ -150,116 +151,19 @@ async function downloadBody(message: ExtendedFailedMessage) {
   }
 
   try {
-    switch (response.headers.get("content-type")) {
-      case "application/json": {
-        const jsonBodyRaw = await response.text();
-        const jsonBody = parse(jsonBodyRaw.replace(/\\"|"(?:\\"|[^"])*"|(\/\/.*|\/\*[\s\S]*?\*\/)/g, (m, g) => (g ? "" : m)));
-        message.messageBody = formatJson(jsonBody);
-        return;
-      }
-      case "text/xml": {
-        const xmlBody = await response.text();
-        message.messageBody = formatXml(xmlBody);
-        return;
-      }
-      default: {
-        message.messageBody = await response.text();
-      }
-    }
-  } catch {
-    message.bodyUnavailable = true;
-  }
-}
-
-// taken from https://github.com/krtnio/angular-pretty-xml/blob/master/src/angular-pretty-xml.js
-function formatXml(xml: string) {
```

Comment:
I found a lib that does the formatting for xml.
I tested a message with xml serialization, and everything works correctly.